tests/check_files/test_check_files/test_system_check_files.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import json
+import os
+import pytest
+from datetime import datetime
+from deepdiff import DeepDiff
+
+OUTPUT_FILE = f"system_checkfiles_{datetime.now().timestamp()}.json"
+
+
+@pytest.fixture
+def get_first_file(request):
+    """Allows to use the --before-file in order to pass the file before an installation, update, or uninstallation.
+
+    Args:
+        request (fixture): Provide information on the executing test function.
+    """
+    return request.config.getoption("--before-file")
+
+
+@pytest.fixture
+def get_second_file(request):
+    """Allows to use the --after-file in order to pass the file after an installation, update, or uninstallation.
+
+    Args:
+        request (fixture): Provide information on the executing test function.
+    """
+    return request.config.getoption("--after-file")
+
+
+@pytest.fixture
+def get_output_path(request):
+    """Allows to use the --output-path so the users can save the output if they want.
+
+    Args:
+        request (fixture): Provide information on the executing test function.
+    """
+    return request.config.getoption("--output-path")
+
+
+def test_system_check_files(get_first_file, get_second_file, get_output_path):
+    """This test checks if two files are not equal.
+
+    After an installation, update or uninstallation is necessary to check if the system files are the same as before.
+    Two given files are checked for differences between them. After checking their contents, if there are differences
+    between them, a log will be printed in JSON format. If the user specifies an output path, it will be saved there
```

Comment:
```suggestion
    between them, the test will fail and a log specifying the differences will be printed in JSON format. If the user specifies an output path, it will be saved there
```

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/2246/commits/1126de5ecc83eecbbc8d9271ae065b3b8420a94c